src/main/java/picard/vcf/LiftoverVcf.java

```diff
@@ -201,15 +204,16 @@ public static void main(final String[] args) {
 
         ProgressLogger progress = new ProgressLogger(log, 1000000, "read");
         // a mapping from original allele to reverse complemented allele
-        final Map<Allele, Allele> reverseComplementAlleleMap = new HashMap<Allele, Allele>(10);
+        final Map<Allele, Allele> reverseComplementAlleleMap = new HashMap<>(10);
 
         for (final VariantContext ctx : in) {
             ++total;
             final Interval source = new Interval(ctx.getContig(), ctx.getStart(), ctx.getEnd(), false, ctx.getContig() + ":" + ctx.getStart() + "-" + ctx.getEnd());
             final Interval target = liftOver.liftOver(source, LIFTOVER_MIN_MATCH);
 
-            // if the target is null OR (the target is reverse complemented AND the variant is an indel or mixed), then we cannot lift it over
-            if (target == null || (target.isNegativeStrand() && (ctx.isMixed() || ctx.isIndel()))) {
+            // if the target is null OR (the target is reverse complemented AND the variant is a non-biallelic indel or mixed), then we cannot lift it over
+
+            if (target == null || (target.isNegativeStrand() && (ctx.isMixed() || ctx.isIndel() && !ctx.isBiallelic()))) {
```

Comment:
could you please add parentheses around the final terms -- just for readability so it's clear where the ORs and ANDs apply?